### PR TITLE
feat(st-tip): New component Tip

### PR DESCRIPTION
**Pull request by jvelaystratio**
## TYPE
- New component

## Description
Add a new component tip

EGE-331

### Documentation
- [ ] Have explained API (Inputs, Outpus, Models, Functions, etc)
- [x] Have example running
- [x] Have example code (HTML, ts)
- [x] Have changelog.md updated

### Code
- [x] Have 3 spaces indentation
- [x] Pass ts lint Task
- [x] Pass Sass lint Task
- [x] Have QA-tags

### Tests
- [x] Have 70% Tests coverage

### Other observations
- Add other information you consider important

### Post-review reminder
- Remember reviewer approve changes inside the Files Changed tab (Review changes dropdown)
- Squash commits
- Release version in case of breaking changes and notify of all front

### CODE REVIEW

File: src/lib/st-tip/st-tip.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      })
+         .compileComponents();  // compile template and css
+   }));
+
+   beforeEach(() => {
+      fixture = TestBed.createComponent(StTipComponent);
+      component = fixture.componentInstance;
+   });
+
+   it('It should have requiered inputs defined', () => {
```
Comment (EvaMGC) [inline src/lib/st-tip/st-tip.component.spec.ts]: Change requiered  to required

File: src/lib/st-tip/st-tip.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      component.text = text;
+      fixture.detectChanges();
+      expect(fixture.componentInstance.theme).toEqual('themeA');
+
+      component.theme = 'themeB';
+      fixture.detectChanges();
+      expect(fixture.componentInstance.theme).toEqual('themeB');
+   });
+
+   it('If themeA is defined, need to be applied', () => {
```
Comment (EvaMGC) [inline src/lib/st-tip/st-tip.component.spec.ts]: Change need  to 'it needs'

File: src/lib/st-tip/st-tip.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+      component.qaTag = qaTag;
+      component.text = text;
+
+      component.theme = 'themeA';
+      fixture.detectChanges();
+      el = fixture.debugElement.query(By.css('.themeA'));
+      expect(el.nativeElement).toBeDefined();
+   });
+
+   it('If themeB is defined, need to be applied', () => {
```
Comment (EvaMGC) [inline src/lib/st-tip/st-tip.component.spec.ts]: Change need to 'it needs'

File: src/lib/st-tip/st-tip.component.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+@StEgeo()
+@Component({
+   selector: 'st-tip',
+   templateUrl: './st-tip.component.html',
+   styleUrls: ['./st-tip.component.scss'],
+   changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class StTipComponent {
+
+  @Input() @StRequired() qaTag: string;
```
Comment (ppenalver) [inline src/lib/st-tip/st-tip.component.ts]: qaTag now its not required

**Pull request merged**